lib/Teng/Plugin/SearchJoined.pm

```diff
@@ -22,15 +22,9 @@ sub search_joined {
     }
     my @tables = map { $self->{schema}->get_table($_) } @table_names;
 
-    my $name_sep = $self->{sql_builder}{name_sep};
-    my @fields;
-    for my $table (@tables) {
-        my $table_name = $table->name;
-        my @columns = map { "$table_name$name_sep$_" } @{ $table->columns };
-        push @fields, @columns;
-    }
+    my $fields = _get_select_joined_columns($self, \@tables, $opt);
```

Comment:
updated: https://github.com/kfly8/p5-Teng-Plugin-SearchJoined/blob/columns_opts/lib/Teng/Plugin/SearchJoined.pm#L25

But export `_get_select_joined_columns` : https://github.com/kfly8/p5-Teng-Plugin-SearchJoined/blob/columns_opts/lib/Teng/Plugin/SearchJoined.pm#L12

Is it ok..?